src/configure

```diff
@@ -184,6 +189,29 @@ do
   --cudatk-dir=*)
     CUDATKDIR=`read_dirname $1`;
     shift ;; #CUDA is used in src/cudamatrix and src/nnet{,bin} only
+  --host=*)
```

Comment:
If you want to wait on this set until #1249 merges, I can rebase it and use that instead.  But doing it with the --host flag allows the user to specify the host once, instead of 5 times (for all 5 tools used).